images/kubernetesonarm/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM larmog/armhf-alpine-java:jdk-8u73
+
+EXPOSE 9200 9300
```

Comment:
But it tells your users of your image that it expects these ports to be exposed. If you are looking for optimization then perhaps a better solution is to flatten you image after build.